package.json

```diff
@@ -50,6 +51,117 @@
     "chai": "^3.5.0",
     "vscode-textmate": "^2.1.1"
   },
+  "runtimeDependencies": [
```

Comment:
It is what it is. :smile: Before this change, acquisition of OmniSharp was deferred until the server launched. So, we could acquire just the necessary OmniSharp server at that time. Now that all acquisition happens up front, both servers get acquired at once.

In the near, I'll be removing the CoreCLR version of OmniSharp completely. So, this is a point in time problem.